elements/pfe-card/src/pfe-card.scss

```diff
@@ -147,59 +151,66 @@ $LOCAL: card;
 }
 
 ::slotted([slot="pfe-card--header"]) {
-  display: block;
   z-index: 2;
   // Declare the header background color
-  background-color: #{pfe-local($cssvar: BackgroundColor, $region: header)};
-  color: #{pfe-local($cssvar: Color, $region: header)};
+  background-color: pfe-local($cssvar: BackgroundColor, $region: header);
+  color:            pfe-local($cssvar: Color, $region: header);
 
   // Margin style only on top and sides
   margin-top:    -2rem; //IE11 fallback
-  margin-top:    calc(-1 * #{pfe-local(PaddingTop)}) !important;
+  margin-top:    calc(#{pfe-local(PaddingTop)} * -1) !important;
   margin-right:  -2rem; //IE11 fallback
-  margin-right:  calc(-1 * #{pfe-local(PaddingRight)});
-  margin-bottom: #{pfe-local(spacing)};
+  margin-right:  calc(#{pfe-local(PaddingRight)} * -1);
+  margin-bottom: pfe-local(spacing--vertical);
   margin-left:   -2rem; //IE11 fallback
-  margin-left:   calc(-1 * #{pfe-local(PaddingLeft)});
+  margin-left:   calc(#{pfe-local(PaddingLeft)} * -1);
 
   // Padding for the header region
-  padding-top:    #{pfe-local(spacing)}; // Top padding differs if background color applied
-  padding-right:  #{pfe-local(PaddingRight)};
-  padding-left:   #{pfe-local(PaddingLeft)};
-  padding-bottom: #{pfe-local(spacing)};
+  padding-top:    pfe-local(spacing--vertical); // Top padding differs if background color applied
+  padding-right:  pfe-local(PaddingRight);
+  padding-left:   pfe-local(PaddingLeft);
+  padding-bottom: pfe-local(spacing--vertical);
 }
 
 ::slotted(:not([slot])) {
-  display: block;
-  margin-bottom: #{pfe-local(spacing)};
+  margin-bottom: #{pfe-local(spacing--vertical)};
 }
 
-// Flexbox
-// Fix: https://github.com/patternfly/patternfly-elements/issues/478
-[name="pfe-card--footer"] {
-  display: flex;
-  flex-flow: row wrap;
-  margin-bottom: calc(calc(#{pfe-local(spacing)} / 2) * -1);
-  margin-left: calc(calc(#{pfe-local(spacing)} / 2) * -1);
-  margin-right: calc(calc(#{pfe-local(spacing)} / 2) * -1);
-  align-items: baseline; // Make buttons and CTAs line up
+// Targets the slot in the light DOM
+::slotted([slot="pfe-card--footer"]:not([pfe-overflow~="bottom"])) {
```

Comment:
There was a bug adding the margins to the footer region with how it interacted with the overflow attribute so I wrapped them in these not selectors :D